services/core-web/src/components/mine/MineHeader.js

```diff
@@ -284,20 +279,23 @@ export class MineHeader extends Component {
                   trigger="click"
                 >
                   <Button ghost style={{ padding: 0, margin: 0, height: 0 }}>
-                    View Notes{" "}
-                    <img
-                      alt="info"
-                      className="padding-small"
-                      src={INFO_CIRCLE}
-                      style={{ padding: 0, margin: 0 }}
-                    />
+                    View Notes <CoreTooltip />
                   </Button>
                 </Popover>
               ) : (
                 <p>{String.EMPTY_FIELD}</p>
               )}
             </div>
           </div>
+          <div className="inline-flex padding-small wrap">
+            <p className="field-title">Exemption Status</p>
+            <div>
+              {(null && this.props.mine.exemption_fee_status.description) || String.EMPTY_FIELD}
```

Comment:
What's the desired output here? This always displayed `String.EMPTY_FIELD` because of the `null &&`.

Reply:
This was test code I forgot to undo. Updated